### ISSUE TITLE
Error an illegal memory access was encountered at line 184 in file C:\bnb\wt2\csrc\ops.cu

### ISSUE THREAD

**Issue by theseraphim** [d8ahazard/sd_dreambooth_extension#915]
### Kindly read the entire form below and fill it out with the requested information.

**Please find the following lines in the console and paste them below. _If you do not provide this information, your
issue will be automatically closed._**

`
venv "I:\stable-diffusion-webui\venv\Scripts\Python.exe"
Python 3.10.9 | packaged by conda-forge | (main, Jan 11 2023, 15:15:40) [MSC v.1916 64 bit (AMD64)]
Commit hash: ea9bd9fc7409109adcd61b897abc2c8881161256
Installing requirements for Web UI

#######################################################################################################
Initializing Dreambooth
If submitting an issue on github, please provide the below text for debugging purposes:

Python revision: 3.10.9 | packaged by conda-forge | (main, Jan 11 2023, 15:15:40) [MSC v.1916 64 bit (AMD64)]
Dreambooth revision: 21ddebabb47274255d516e655f997d9ddadc0fe4
SD-WebUI revision: ea9bd9fc7409109adcd61b897abc2c8881161256

Checking Dreambooth requirements...
[+] bitsandbytes version 0.35.0 installed.
[+] diffusers version 0.10.2 installed.
[+] transformers version 4.25.1 installed.
[+] xformers version 0.0.16 installed.
[+] torch version 1.13.1+cu117 installed.
[+] torchvision version 0.14.1+cu117 installed.

#######################################################################################################
`

**Have you read the [Readme](https://github.com/d8ahazard/sd_dreambooth_extension#readme)?** Yes

**Have you completely restarted the stable-diffusion-webUI, not just reloaded the UI?** Yes

**Have you updated Dreambooth to the latest revision?** Yes

**Have you updated the Stable-Diffusion-WebUI to the latest version?** Yes

**No, really. Please save us both some trouble and update the SD-WebUI and Extension and restart before posting this.
Reply 'OK' Below to acknowledge that you did this.** OK

**Describe the bug**

Intermittent error during training, 

"Error an illegal memory access was encountered at line 184 in file C:\bnb\wt2\csrc\ops.cu"

theres no python error and the whole thing just blarps and dies with a "press any key to continue"

**Provide logs**
There is no stack trace, so ive included what ive got (that file it references btw, does not exist)

`Initializing dreambooth training...
Pre-processing lexa: 100%|█████████████████████████████████████████████████████████████| 8/8 [00:00<00:00, 1999.91it/s]
Pre-processing classifiers_0: 100%|███████████████████████████████████████████████████| 40/40 [00:00<00:00, 182.40it/s]
Concept requires 5 class images per instance image.
Sorting instance images: 100%|███████████████████████████████████████████████████████████████████| 1/1 [00:00<?, ?it/s]
Nothing to generate.
Replace CrossAttention.forward to use xformers
Found 40 reg images.
Preparing dataset...
Init dataset!
Preparing Dataset (With Caching)
Bucket 0 (512, 512, 0) - Instance Images: 8 | Class Images: 40 | Max Examples/batch: 16
Total Buckets 1 - Instance Images: 8 | Class Images: 40 | Max Examples/batch: 16
Caching latents...: 100%|██████████████████████████████████████████████████████████████| 48/48 [00:12<00:00,  4.27it/s]
Total images / batch: 16, total examples: 16
Caching latents...: 100%|██████████████████████████████████████████████████████████████| 48/48 [00:12<00:00,  3.96it/s]
Total dataset length (steps): 16
Initializing bucket counter!
  ***** Running training *****
  Num batches each epoch = 16
  Num Epochs = 100
  Batch Size Per Device = 1
  Gradient Accumulation steps = 1
  Total train batch size (w. parallel, distributed & accumulation) = 1
  Text Encoder Epochs: 75
  Total optimization steps = 800
  Total training steps = 1600
  Resuming from checkpoint: False
  First resume epoch: 0
  First resume step: 0
  Lora: True, Adam: True, Prec: fp16
  Gradient Checkpointing: True
  EMA: False
  UNET: True
  Freeze CLIP Normalization Layers: False
  LR: 0.0002
  LoRA Text Encoder LR: 0.0002
  V2: False
Steps:  11%|█▋              | 170/1600 [03:53<32:30,  1.36s/it, loss=0.403, loss_avg=0.0747, lr=0.0002, vram_usage=6.1]Error an illegal memory access was encountered at line 184 in file C:\bnb\wt2\csrc\ops.cu
Press any key to continue . . .

(base) I:\stable-diffusion-webui>webui.bat`

**Environment**
What OS?
Windows 11
If Windows - WSL or native?
Native (miniconda)
What GPU are you using?
nvidia gtx 1070 8GB 

**Screenshots/Config**

If the issue is specific to an error while training, please provide a screenshot of training parameters or the
db_config.json file from /models/dreambooth/MODELNAME/db_config.json

`{
    "adamw_weight_decay": 0.01,
    "attention": "xformers",
    "cache_latents": true,
    "center_crop": false,
    "freeze_clip_normalization": false,
    "clip_skip": 1,
    "concepts_list": [
        {
            "class_data_dir": "",
            "class_guidance_scale": 7.5,
            "class_infer_steps": 40,
            "class_negative_prompt": "((((ugly)))), (((duplicate))), ((morbid)), ((mutilated)), out of frame, extra fingers, mutated hands, ((poorly drawn hands)), ((poorly drawn face)), (((mutation))), (((deformed))), ((ugly)), blurry, ((bad anatomy)), (((bad proportions))), ((extra limbs)), cloned face, (((disfigured))), out of frame, ugly, extra limbs, (bad anatomy), gross proportions, (malformed limbs), ((missing arms)), ((missing legs)), (((extra arms))), (((extra legs))), mutated hands, (fused fingers), (too many fingers), (((long neck)))",
            "class_prompt": "a photo of a womans face",
            "class_token": "",
            "instance_data_dir": "C:\\Users\\david\\Downloads\\lexa",
            "instance_prompt": "lexadoig",
            "instance_token": "",
            "is_valid": true,
            "n_save_sample": 1,
            "num_class_images": 0,
            "num_class_images_per": 5,
            "sample_seed": -1,
            "save_guidance_scale": 7.5,
            "save_infer_steps": 40,
            "save_sample_negative_prompt": "",
            "save_sample_prompt": "a photo of lexadoig",
            "save_sample_template": ""
        }
    ],
    "concepts_path": "",
    "custom_model_name": "",
    "epoch": 0,
    "epoch_pause_frequency": 0.0,
    "epoch_pause_time": 0.0,
    "gradient_accumulation_steps": 1,
    "gradient_checkpointing": true,
    "gradient_set_to_none": false,
    "graph_smoothing": 50.0,
    "half_model": false,
    "train_unfrozen": false,
    "has_ema": false,
    "hflip": true,
    "initial_revision": 0,
    "learning_rate": 2e-06,
    "learning_rate_min": 1e-06,
    "lifetime_revision": 0,
    "lora_learning_rate": 0.0002,
    "lora_model_name": [],
    "lora_rank": 32,
    "lora_txt_learning_rate": 0.0002,
    "lora_txt_weight": 1,
    "lora_weight": 1,
    "lr_cycles": 1,
    "lr_factor": 0.5,
    "lr_power": 1,
    "lr_scale_pos": 0.5,
    "lr_scheduler": "constant_with_warmup",
    "lr_warmup_steps": 0,
    "max_token_length": 75,
    "mixed_precision": "fp16",
    "model_name": "lexa",
    "model_dir": "I:\\stable-diffusion-webui\\models\\dreambooth\\lexa",
    "model_path": "I:\\stable-diffusion-webui\\models\\dreambooth\\lexa",
    "num_train_epochs": 100,
    "pad_tokens": true,
    "pretrained_model_name_or_path": "I:\\stable-diffusion-webui\\models\\dreambooth\\lexa\\working",
    "pretrained_vae_name_or_path": "",
    "prior_loss_scale": false,
    "prior_loss_target": 100.0,
    "prior_loss_weight": 0.75,
    "prior_loss_weight_min": 0.1,
    "resolution": 512,
    "revision": 0,
    "sample_batch_size": 1,
    "sanity_prompt": "",
    "sanity_seed": 420420.0,
    "save_ckpt_after": true,
    "save_ckpt_cancel": false,
    "save_ckpt_during": false,
    "save_embedding_every": 20,
    "save_lora_after": true,
    "save_lora_cancel": true,
    "save_lora_during": true,
    "save_preview_every": 20,
    "save_safetensors": false,
    "save_state_after": false,
    "save_state_cancel": false,
    "save_state_during": false,
    "scheduler": "ddim",
    "shuffle_tags": true,
    "snapshot": "",
    "src": "I:\\stable-diffusion-webui\\models\\Stable-diffusion\\v1-5-pruned.ckpt",
    "stop_text_encoder": 0.75,
    "train_batch_size": 1,
    "train_imagic": false,
    "train_unet": true,
    "use_8bit_adam": true,
    "use_concepts": false,
    "use_ema": false,
    "use_lora": true,
    "use_subdir": true,
    "v2": false
}`

The error happens with different attempts at LoRa training, no seeming pattern to it, just kaputs sometimes, sometimes does not.
GPU is stable, not OCd, cool (62C at heavy load)

**Comment by theseraphim** [comment]
never mind, it only comes up if i also try to watch a video... something in the process is causing a memory error.
no youtube, no error.
